Week 1/DeMarcus/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Demarcus McMillian the dev</title>
+</head>
+<body>
+    <img src="jpg-file.jpg" style="width:128px;height:128px;">
```

Comment:
You should organize your code to flow, add an image directory for all your image